darwin/dataset/remote_dataset.py

```diff
@@ -330,6 +339,97 @@ def get_release(self, name: str = "latest"):
                 return release
         raise NotFound(self.identifier)
 
+    def split(
+        self,
+        val_percentage: float = 0.1,
+        test_percentage: float = 0,
+        force_resplit: bool = False,
+        split_seed: int = 0,
+        make_default_split: bool = True,
+    ):
+        """
+        Creates lists of file names for each split for train, validation, and test.
+        Note: This functions needs a local copy of the dataset
+
+        Parameters
+        ----------
+        val_percentage : float
+            Percentage of images used in the validation set
+        test_percentage : float
+            Percentage of images used in the test set
+        force_resplit : bool
+            Discard previous split and create a new one
+        split_seed : int
+            Fix seed for random split creation
+        make_default_split: bool
+            Makes this split the default split
+        """
+        if not self.local_path.exists():
+            raise NotFound(
+                "Local dataset not foungd: the split is perfomed on the local copy of the dataset. \
+                           Pull the dataset from Darwin first using pull()"
+            )
+        split_dataset(
+            self.local_path,
+            val_percentage=val_percentage,
+            test_percentage=test_percentage,
+            force_resplit=force_resplit,
+            split_seed=split_seed,
+            make_default_split=make_default_split,
+        )
+
+    def classes(self, annotation_type: str):
+        """
+        Returns the list of `class_type` classes
+
+        Parameters
+        ----------
+        annotation_type
+            The type of annotation classes, e.g. 'tag' or 'polygon'
+
+        Returns
+        -------
+        classes: list
+            List of classes in the dataset of type `class_type`
+        """
+        assert self.local_path.exists()
```

Comment:
don't really want to crash here? rather raise an error or re-generate the class list.